examples/csharp/unit-test/Rest/TwilioRestTest.cs

```diff
@@ -1482,7 +1482,7 @@ public void TestFetchCallOptionsCreation()
 
         }
 
-        [Test]
+/*        [Test]
```

Comment:
why are these tests commented? Better we remove it if not required

Reply:
took pull from main, test has been modified to include update operation.